tests/director/Makefile

```diff
@@ -44,6 +44,19 @@ sandbox-test:
 run:
 	@kubectl exec -n kyma-system oct-tp-compass-e2e-tests-compass-e2e-director-local-0 -c director-tests -- ./director.test -test.run $(testName) -test.v
 
+
+run-application:
+	@kubectl exec -n kyma-system oct-tp-compass-e2e-tests-compass-e2e-director-local-0 -c director-tests -- ./director-application.test -test.run $(testName) -test.v
+
+run-formation:
+	@kubectl exec -n kyma-system oct-tp-compass-e2e-tests-compass-e2e-director-local-0 -c director-tests -- ./director-formation.test -test.run $(testName) -test.v
+
+run-runtime:
+	@kubectl exec -n kyma-system oct-tp-compass-e2e-tests-compass-e2e-director-local-0 -c director-tests -- ./director-formation.test -test.run $(testName) -test.v
```

Comment:
Copy+paste error. Instead of `-- ./director-formation.test` it should be `-- ./director-runtime.test`.